burr/core/application.py

```diff
@@ -867,24 +874,28 @@ def callback(
                 result: Optional[dict],
                 state: State,
                 exc: Optional[Exception] = None,
-                seq_id=self.sequence_id,
+                # seq_id=self.sequence_id,
```

Comment:
this supposed to be commented?

Reply:
delete otherwise?

Reply:
Yep, not meant to be there (it made things more complicated)